api/src/paths/project/{projectId}/survey/{surveyId}/upload.ts

```diff
@@ -0,0 +1,104 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { SYSTEM_ROLE } from '../../../../../constants/roles';
+import { getDBConnection } from '../../../../../database/db';
+import { HTTP400 } from '../../../../../errors/custom-error';
+import { authorizeRequestHandler } from '../../../../../request-handlers/security/authorization';
+import { PlatformService } from '../../../../../services/platform-service';
+import { getLogger } from '../../../../../utils/logger';
+
+const defaultLog = getLogger('/api/project/{projectId}/survey/{surveyId}/upload');
+
+export const POST: Operation = [
+  authorizeRequestHandler((req) => {
+    return {
+      and: [
+        {
+          validSystemRoles: [SYSTEM_ROLE.SYSTEM_ADMIN],
+          projectId: Number(req.params.projectId),
+          discriminator: 'SystemRole'
+        }
+      ]
+    };
+  }),
+  uploadSurveyDataToBioHub()
+];
+
+POST.apiDoc = {
+  description: 'upload survey/observation data to BioHub.',
+  tags: ['survey'],
+  security: [
+    {
+      Bearer: []
+    }
+  ],
+  parameters: [
+    {
+      in: 'path',
+      name: 'projectId',
+      schema: {
+        type: 'integer',
+        minimum: 1
+      },
+      required: true
+    },
+    {
+      in: 'path',
+      name: 'surveyId',
+      schema: {
+        type: 'integer',
+        minimum: 1
+      },
+      required: true
+    }
+  ],
+  responses: {
+    200: {
+      description: 'Boolean true value representing successful deletion.',
+      content: {
+        'application/json': {
+          schema: {
+            title: 'Survey delete response',
+            type: 'boolean'
+          }
+        }
+      }
```

Comment:
Can just remove this content section entirely, and then for the response just say `return res.status(200).send();`

Reply:
Maybe update the description too